package.json

```diff
@@ -81,5 +81,8 @@
     "{src,test}/**/*.{js,jsx}": [
       "eslint"
     ]
+  },
+  "resolutions": {
+    "scheduler": "0.15.0"
```

Comment:
Can you add a comment telling why we need to do that ?

Reply:
Commit message talks for itself, no?

Reply:
Ah, it seems I missed the description of the commit, my bad. I still think it is easier to have this kind of information directly in the form of a comment, but now that I think about it, since JSON does not support comments, it will be hard. OK for me :)